internal/pkg/service/buffer/dependencies/mocked.go

```diff
@@ -144,7 +144,7 @@ func (v *mocked) StatsCacheNode() *statistics.CacheNode {
 
 func (v *mocked) EventSender() *event.Sender {
 	if v.eventSender == nil {
-		v.eventSender = event.NewSender(v.Logger(), v.KeboolaProjectAPI())
+		v.eventSender = event.NewSender(v.Logger())
```

Comment:
S posielanim eventov bolo viacero problemov.

Prvy commit je od @jprochazk , z https://github.com/keboola/keboola-as-code/pull/1207

^^^ tam sme neboli schopny doriesit testy a cakalo to na vylepsenie testov, rebasol som to do tohto PR a priadl nejake dalsie veci co som nasiel.